app/javascript/packages/webauthn/webauthn-input-element.ts

```diff
@@ -19,11 +18,10 @@ export class WebauthnInputElement extends HTMLElement {
       return;
     }
 
-    if (isWebauthnPasskeySupported() && (await isWebauthnPlatformAuthenticatorAvailable())) {
```

Comment:
I wouldn't expect to see this code entirely removed, since we still only want to allow devices considered unsupported by `isWebauthnPasskeySupported()` if the user is part of the A/B test.

Reply:
> I wouldn't expect to see this code entirely removed, since we still only want to allow devices considered unsupported by `isWebauthnPasskeySupported()` if the user is part of the A/B test.

I might be confused but shouldn't a user without a device be an automatic disqualifier for the test?

Reply:
What I mean is that if we remove the `isWebauthnPasskeySupported()`, then "Face or touch unlock" will be visible as an option to _everyone_ (all desktop users), not just those who are selected into the A/B test.

Reply:
Ah, gotcha. So the line will be 
```
// if webauthn passkeys  are available, supported, and a part of the A/B test
```
Okay cool.

Reply:
More precisely, it'd be something like:

```ts
if (((isWebauthnPasskeySupported() || isOptedInToAbTest())  && (await isWebauthnPlatformAuthenticatorAvailable())) {
  // ...
}
```

Reply:
Then from that, you could still get if supported and available, then show. What if we changed this up a little bit?

```
if (((isWebauthnPasskeySupported() || isOptedInToAbTest()) && (await isWebauthnPlatformAuthenticatorAvailable())) 
{ 
// ... 
}
```

Because then if its not available _at all_, then we can skip over the second check which would automatically be false?